components/raftstore/src/store/config.rs

```diff
@@ -386,6 +387,7 @@ impl Default for Config {
             hibernate_regions: true,
             dev_assert: false,
             apply_yield_duration: ReadableDuration::millis(500),
+            apply_yield_msg_size: ReadableSize::kb(32),
```

Comment:
In my test, 32kb is a good choice because the max message thread hold is 16kb and apply thread will flush the write batch at 256 keys. In the general case, one batch entry won't be too large when there is no bottleneck; but when the process reaches it bottleneck, big raft entry should result in better overall throughput.

Reply:
Then you need to explain the configuration in docs and code comment why it's smaller than entry size by default.